skaffold.yaml.template

```diff
@@ -15,6 +15,7 @@ deploy:
     flags:
       upgrade:
         - --install
+        - --debug
```

Comment:
Did you mean to leave this in?

Reply:
yes it spits out what is actually being deployed (raw kubernetes yamls), I think it is helpful see snippet below
```
---
# Source: datarepo/charts/oidc-proxy/templates/oidc-ingress.yaml
apiVersion: extensions/v1beta1
kind: Ingress
metadata:
  name: ms-jade-oidc-proxy
  labels:
    helm.sh/chart: oidc-proxy-0.0.3
    app.kubernetes.io/name: oidc-proxy
    app.kubernetes.io/instance: ms-jade
    app.kubernetes.io/managed-by: Helm
  annotations:
    kubernetes.io/ingress.global-static-ip-name: "jade-dev-ms"
    kubernetes.io/ingress.allow-http: "false"
spec:
  tls:
    - secretName: oidc-tls-key
  rules:
  - http:
      paths:
      - path: /*
        backend:
          serviceName: ms-jade-oidc-proxy
          servicePort: http
```